buildozer/targets/android.py

```diff
@@ -97,6 +104,47 @@ def _p4a(self, cmd, **kwargs):
         kwargs.setdefault('cwd', self.pa_dir)
         return self.buildozer.cmd(self._p4a_cmd + cmd + self.extra_p4a_args, **kwargs)
 
+    @property
+    def p4a_best_android_ndk(self):
+        """
+        Return the p4a's recommended android's NDK version, depending on the
+        p4a version used for our buildozer build. In case that we don't find
+        it, we will return the buildozer's recommended one, defined by global
+        variable `ANDROID_NDK_VERSION`.
+        """
+        if self.p4a_best_ndk_version is not None:
+            # make sure to read p4a version only the first time
+            return self.p4a_best_ndk_version
+
+        if not hasattr(self, "pa_dir"):
+            pa_dir = join(self.buildozer.platform_dir, self.p4a_directory)
+        else:
+            pa_dir = self.pa_dir
+
+        # check p4a's recommendation file, and in case that exists find the
+        # recommended android's NDK version, otherwise return buildozer's one
+        ndk_version = ANDROID_NDK_VERSION
+        rec_file = join(pa_dir, "pythonforandroid", "recommendations.py")
+        if (not os.path.exists(pa_dir)) or (not os.path.isfile(rec_file)):
```

Comment:
Minor: why not only checking for `os.path.isfile(rec_file)`?

Reply:
Thanks @AndreMiras :smile: